README.md

```diff
@@ -283,6 +283,27 @@ scss-lint --format=XML [scss-files...]
 </lint>
 ```
 
+### CheckStyle
+
+Outputs a CheckStyle XML with `<checkstyle>`, `<file>`, and `<error>` tags. Suitable for
+consumption by tools like [Jenkins](http://jenkins-ci.org/) with [CheckStyle plugin](https://wiki.jenkins-ci.org/display/JENKINS/Checkstyle+Plugin).
+
+```bash
+scss-lint --format=CheckStyle [scss-files...]
+```
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<checkstyle version="1.5.6">
```

Comment:
Including the version number here seems brittle. Is this absolutely necessary (i.e. does the CheckStyle plugin require this attribute be present)?

Having `scss-lint` need to update this formatter's output each time the `CheckStyle` plugin is updated isn't ideal.